docs/examples/control/example_camera_config.rst

```diff
@@ -0,0 +1,33 @@
+*******************************************************************************
+Camera Config
+*******************************************************************************
+
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+Default Camera Config
+====================
```

Comment:
line needs to have at least length of text

Reply:
Thanks for the review!

Yes... noticed. my font is not monospace so sometimes my eyes cheat me... this push should be good. no error building the doc locally.